README.md

```diff
@@ -3,27 +3,54 @@ webcomponents.js
 
 [![Build Status](https://travis-ci.org/webcomponents/webcomponentsjs.svg?branch=master)](https://travis-ci.org/webcomponents/webcomponentsjs)
 
-A suite of polyfills supporting the [Web Components](http://webcomponents.org) specs.
+A suite of polyfills supporting the [Web Components](http://webcomponents.org) specs:
 
 - **Custom Elements**: allows authors to define their own custom tags ([spec](https://w3c.github.io/webcomponents/spec/custom/)).
 - **HTML Imports**: a way to include and reuse HTML documents via other HTML documents ([spec](https://w3c.github.io/webcomponents/spec/imports/)).
 - **Shadow DOM**: provides encapsulation by hiding DOM subtrees under shadow roots ([spec](https://w3c.github.io/webcomponents/spec/shadow/)).
 
-## Releases
+## How to use
 
-Pre-built (concatenated & minified) versions of the polyfills are maintained in the [tagged versions](https://github.com/webcomponents/webcomponentsjs/releases) of this repo. There are several variants:
+The polyfills are built (concatenated & minified) into several bundles that target
+different browsers and spec readiness:
 
-- `webcomponents-lite.js` includes all of the polyfills.
-- `webcomponents-loader.js` is a custom loader that dynamically load a minified polyfill
-bundle, using feature detection. The bundles that can be loaded are:
-  - `webcomponents-hi` -- HTML Imports (needed by Safari Tech Preview)
-  - `webcomponents-hi-ce` -- HTML Imports and Custom Elements (needed by Safari 10)
-  - `webcomponents-hi-ce-sd` -- HTML Imports, Custom Elements and Shady DOM/CSS (needed by Safari 9, Firefox, Edge)
-  - `webcomponents-lite` -- HTML Imports, Custom Elements, Shady DOM/CSS and generic platform polyfills (such as Template, ES6 Promise, Constructable events, etc.) (needed by Internet Explorer 11)
+- `webcomponents-hi` -- HTML Imports (needed by Safari Tech Preview)
+- `webcomponents-hi-ce` -- HTML Imports and Custom Elements (needed by Safari 10)
+- `webcomponents-hi-sd-ce` -- HTML Imports, Custom Elements and Shady DOM/CSS (needed by Safari 9, Firefox, Edge)
+- `webcomponents-sd-ce` -- Custom Elements and Shady DOM/CSS (no HTML Imports)
+- `webcomponents-lite` -- all of the polyfills: HTML Imports, Custom Elements, Shady DOM/CSS and generic platform polyfills (such as Template, ES6 Promise, Constructable events, etc.) (needed by Internet Explorer 11)
```

Comment:
Yikes, just realized that we might need to tweak loader so that `webcomponents-lite` is loaded for Edge too -- let's double-check tomorrow.  Technically Edge has Template but their [impl is pretty busted](https://github.com/webcomponents/template/commit/b6137231f28449bb66837beac82145b0ac9c069e) (we fix it up in the Template polyfill).  Since most testing is using `webcomponents-lite` this hasn't come up in tests.  😩